
# general
depth_image_topic: /XTION/depth/image
camera_info_topic: /XTION/depth/camera_info

# 100 is more or less the limit to track
# with threshold: 0.75 and sigmas at 0.0005

downsampling: 32.0 # wrt VGA resolution

# VirtualObject 
object_package: pose_tracking
object_model: /object/calib_obj.obj
#object_package: arm_object_models
#object_model: /objects/impact_battery/impact_battery.obj
pose_x: 0.0
pose_y: 0.09
pose_z: 0.4
pose_alpha: 0.325
pose_beta: 0.5
pose_gamma: 0.125
pose_alpha_v: 0.0
pose_beta_v: 0.0
pose_gamma_v: 0.085

# process model
linear_acceleration_sigma:  0.1 #0.1  0.2  1.0 2.0
angular_acceleration_sigma: 1.5 #1.5  3.0  15.0 30.0
damping: 5.0 #5.0  20.0  1000.0 1000.0

bg_depth_sim: 7.0
bg_depth: 3.5
bg_sigma: 1.5

# observation model
w_sigma:     0.0005          # UKF 0.005
w_sigma_sim: 0.0005

# parameter process model
A_b: 0.9 # 0.9
v_sigma_b: 0.5 #0.5

error_delay: 10
error_pixels: 1
error_pixel_depth: 0.5
step_through: false

max_cycles: -1
